src/idl_gen_rust.cpp

```diff
@@ -216,6 +216,7 @@ bool GenerateRustModuleRootFile(const Parser &parser,
       for (auto it = sub_modules.begin(); it != sub_modules.end(); it++) {
         code += "pub mod " + it->first + " {";
         code.IncrementIdentLevel();
+        code += "#![deny(unsafe_op_in_unsafe_fn)]";
```

Comment:
I think it makes more sense for this to be at the top of the module root file, or at the top of every generated object file.  I believe the `#![...]` attributes will cover all nested modules so there's some redundant coverage and also it currently misses the generated code that's in the root namespace.